back/PowygrywaniApi/PowygrywaniApi/Model/Rating.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PowygrywaniApi.Model
+{
+    public class Rating : ModelTemplate
```

Comment:
`Rating` nie ma mieć `name`, więc albo tutaj bez dziedziczenia albo rozbić to dziedziczenie na 2 klasy. {klasa z id} -> {klasa z name} -> {inne klasy}